modules/templates/helper.go

```diff
@@ -339,33 +339,37 @@ func RenderCommitMessageLink(msg, urlPrefix, urlDefault string, metas map[string
 // the provided default url, handling for special links without email to links.
 func RenderCommitMessageLinkSubject(msg, urlPrefix, urlDefault string, metas map[string]string) template.HTML {
 	cleanMsg := template.HTMLEscapeString(msg)
+
+	msgLines := strings.Split(strings.TrimSpace(string(cleanMsg)), "\n")
+	if len(msgLines) == 0 {
```

Comment:
Is this condition ever true? `strings.Split("", "\n")` returns `[]string{""}` AFAIK, which has a len of 1, so `len(msgLines)` can never by 0.

Reply:
I did not change that, that was how this code was before also ;)

Reply:
> 
> 
> I did not change that, that was how this code was before also ;)

Looks exactly like your own addition. What do you mean you did not change that? 🤔 

![image](https://user-images.githubusercontent.com/18600385/67856846-38000380-faf4-11e9-8ce3-56b020efe1a6.png)

Reply:
Anyway fixed check for empty commit message

Reply:
@guillep2k I moved that code just up, same was there before already: https://github.com/go-gitea/gitea/pull/8745/files#diff-7e18c3b28c00aad8476da1a8231f6864L350

Reply:
> 
> 
> @guillep2k I moved that code just up, same was there before already: https://github.com/go-gitea/gitea/pull/8745/files#diff-7e18c3b28c00aad8476da1a8231f6864L350

Yeah, sorry, I saw that later.